packages/react-liveness/src/components/FaceLivenessDetector/StartLiveness/StartLiveness.tsx

```diff
@@ -66,14 +66,17 @@ export function StartLiveness(props: StartLivenessProps): JSX.Element {
           />
         )}
 
-        <Flex justifyContent="center">
+        <Flex direction="column" justifyContent="center">
```

Comment:
We should follow up and move this to CSS since it can't be overridden with CSS otherwise.